cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -43,6 +43,20 @@ else()
   endif()
 endif()
 
+# disable the default gold linker when an alternative was enabled by the user
+if(ENABLE_LLD OR ENABLE_MOLD)
+  set(ENABLE_GOLD OFF CACHE BOOL "" FORCE)
+elseif(ENABLE_GOLD)
+  include(LDGOLD)
+endif()
+if(ENABLE_LLD)
+  set(ENABLE_MOLD OFF CACHE BOOL "" FORCE)
+  include(LLD)
+elseif(ENABLE_MOLD)
+  set(ENABLE_LLD OFF CACHE BOOL "" FORCE)
+  include(MOLD)
+endif()
```

Comment:
...otherwise nothing is included, no linker detected and falling back to ld
```
-- System type: Linux
---------------------------------------------------- missing linker message ?
-- Check if the system is big endian
```